### ISSUE TITLE
Can lsl listen TCP stream sent from other server (not lsl server)?

### ISSUE THREAD

**Issue by iPsych** [sccn/labstreaminglayer#216]
Can lsl listen TCP stream sent from other server (not lsl server)?
I am working with pylsl and integrating one lsl stream with the other TCP stream.
I can listen with socket function of python, but can I directly receive it in pylsl (then integration will be far more easy)

**Comment by dmedine** [comment]
Unfortunately not. You have to transfer the TCP stream to an LSL outlet 
in a python script (or any other language that is LSL enabled).

**Comment by iPsych** [comment]
Thanks for quick reply.
Is there any example of python wrapper like that?

**Comment by cboulay** [comment]
I don't think so. But it's trivial to find an example of how to read a TCP stream in Python, and this repository has examples on how to create outlets and feed them data. It's up to you to bridge the gap. The only tricky part will be translating the TCP packet payload into a timeseries, and this is something that can only be done by someone who knows your specific TCP protocol (or someone who can reverse engineer it).
